frontend/website/static/js/leaderboard.js

```diff
@@ -0,0 +1,47 @@
+function renderParticipant(participant, rank) {
+  const row = document.createElement("div");
+  row.className = "leaderboard-row";
+  if (participant.length == 2) {
+    const rankCell = document.createElement("span");
+    rankCell.textContent = rank;
+    row.appendChild(rankCell);
+
+    const nameCell = document.createElement("span");
+    nameCell.textContent = participant[0];
+    row.appendChild(nameCell);
+
+    const scoreCell = document.createElement("span");
+    scoreCell.textContent = participant[1];
+    row.appendChild(scoreCell);
+  }
+  return row;
+}
+
+function start() {
+  gapi.client
+    .init({
+      apiKey: "AIzaSyDIFwMr7SPGCLl_o6e4UZKi1q9l8snkUZs"
```

Comment:
We can't use env vars b/c we just copy the static files no prorcessing step. Also this will be exposed in the browser anyways. I've restricted the use of the API KEY to our domain, but open to alternatives.

Reply:
kk that makes sense, thx for explaining